ibmcloud_python_sdk/vpc/baremetal.py

```diff
@@ -1045,40 +1046,40 @@ def disassociate_floating_ip(self, bare_metal_server, interface, fip):
-        """Detach volume from an bare metal server
-
-        :param bare_metal_server: Bare metal server name or ID
-        :type bare_metal_server: str
-        :param attachment: Volume attachement name or ID
-        :type attachment: str
-        """
-        try:
-            bare_metal_server_info = self.get_instance(bare_metal_server)
-            if "errors" in bare_metal_server_info:
-                return bare_metal_server_info
-
-            attachment_info = self.get_bare_metal_server_volume_attachment(
-                bare_metal_server_info["id"], attachment)
-            if "errors" in attachment_info:
-                return attachment_info
-
-            path = ("/v1/bare_metal_servers/{}/volume_attachments/{}"
-                    "?version={}&generation={}".format(
-                        bare_metal_server_info["id"],
-                        attachment_info["id"],
-                        self.cfg["version"],
-                        self.cfg["generation"]))
-
-            data = qw("iaas", "DELETE", path, headers())
-
-            if data["response"].status != 204:
-                return data["data"]
-
-            return resource_deleted()
-
-        except Exception as error:
-            print("Error detaching volume with attachment {} from"
-                  "bare metal server"
-                  " {}. {}".format(attachment, bare_metal_server, error))
-            raise
+    # def detach_volume(self, bare_metal_server, attachment):
+    #     """Detach volume from an bare metal server
+
+    #     :param bare_metal_server: Bare metal server name or ID
+    #     :type bare_metal_server: str
+    #     :param attachment: Volume attachement name or ID
+    #     :type attachment: str
+    #     """
+    #     try:
+    #         bare_metal_server_info = self.get_instance(bare_metal_server)
+    #         if "errors" in bare_metal_server_info:
+    #             return bare_metal_server_info
+
+    #         attachment_info = self.get_server_volume_attachment(
+    #             bare_metal_server_info["id"], attachment)
+    #         if "errors" in attachment_info:
+    #             return attachment_info
+
+    #         path = ("/v1/bare_metal_servers/{}/volume_attachments/{}"
+    #                 "?version={}&generation={}".format(
+    #                     bare_metal_server_info["id"],
+    #                     attachment_info["id"],
+    #                     self.cfg["version"],
+    #                     self.cfg["generation"]))
+
+    #         data = qw("iaas", "DELETE", path, headers())
+
+    #         if data["response"].status != 204:
+    #             return data["data"]
+
+    #         return resource_deleted()
+
+    #     except Exception as error:
+    #         print("Error detaching volume with attachment {} from"
+    #               "bare metal server"
+    #               " {}. {}".format(attachment, bare_metal_server, error))
+    #         raise
```

Comment:
It's better to remove the comment from the code for now.

Reply:
done!